src/cppapi/cppapi.cpp

```diff
@@ -14,6 +14,9 @@ namespace openmc {
 using std::string;
 using std::vector;
 
+//Example of link to another file;
+void init_test_file(py::module& );
```

Comment:
In practice, there should probably just be a header for the other file which is `#include`d here rather than explicitly writing the prototype.